docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing
+
+### Issues, Bugs, Feature or Component Requests
+
+While we currently don't accept issues from people outside the Adevinta Platform to this repo, we
+welcome [discussions and questions here](https://github.com/adevinta/spark-android/discussions/categories/general)!
+
+## If you're part of the Adevinta Platform ⬇️
+
+### Requirements
+
+#### [git-lfs](https://git-lfs.com/)
+
+This repository uses git-lfs to handle [cashapp/paparazzi](https://github.com/cashapp/paparazzi)
+screenshot testing.  
+If you introduce visible changes, you'll likely have to update screenshots files for the tests to
+pass.
+
+```bash
+# Install
+brew install git-lfs
+# Check
+git lfs install
+```
+
+#### [Java 17](https://github.com/adevinta/spark-android/issues/74)
+
+We currently use **AGP 8.1.0** which requires developers to use **JDK 17** on Gradle JDK.
+
+_If you're on macOS, you can install it with [brew](https://formulae.brew.sh/formula/openjdk@17)_
+
+```bash
+brew install openjdk@17
+```
+
+> ℹ️ If you're using a device with [Apple silicon (M1/M2)](https://support.apple.com/en-us/HT211814) then you might need to install
+> a [zulu distribution](https://www.azul.com/downloads/zulu-community/?version=java-17-lts&architecture=x86-64-bit&package=jdk)
+
+or install it directly from **Android Studio**:
+
+![](../art/docs/jdk.png)
```

Comment:
This image shows the wrong version.
I think it would more more useful for users to only have the path of the corresponding setting.

```suggestion
or install it directly from **Android Studio**:  
`File` → `Settings` → `Build, Execution, Deployment` → `Build Tools` → `Gradle` → `Gradle JDK`
```

Reply:
Yeah, it's way better 👍